schemas/assessment-schema-1.1.json

```diff
@@ -0,0 +1,150 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://raw.githubusercontent.com/oamg/convert2rhel/main/schemas/assessment-schema-1.0.json",
+    "title": "Convert2rhel Assessment Schema",
+    "description": "Convert2rhel analyzes the system to determine suitability for conversions before it actually starts to convert the system.  This schema defines the format that would be used.",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "actions": {
+            "type": "object",
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[A-Z0-9_]+$": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "messages": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/$defs/action_message"
+                            }
+                        },
+                        "result": {
+                            "$ref": "#/$defs/action_result"
+                        }
+                    }
+                }
+            }
+        },
+        "format_version": {
+            "description": "Constant value that tells us the format of this file.",
+            "const": "1.0"
+        },
+        "status": {
+            "description": "The highest severity of all actions.",
```

Comment:
Maybe not necessary, but I would include _"between messages and results"_ in this description